src/commands/translation_status.rs

```diff
@@ -0,0 +1,81 @@
+pub async fn translation_status(config: &fpm::Config) -> fpm::Result<()> {
+    // it can be original package or translation
+    let translation_status = if let Some(ref original) = config.package.translation_of.as_ref() {
+        translation_package_status(config, original).await?
+    } else if !config.package.translations.is_empty() {
+        original_package_status().await?
+    } else {
+        return Err(fpm::Error::UsageError {
+            message: "neither translation_of nor translations is set".to_string(),
```

Comment:
This is not usage error. Usage error means the person has specified some wrong argument on command line. This is a issue with configuration so use ConfigurationError.